getting-started/creating-hypertables.md

```diff
@@ -30,14 +30,14 @@ Next, transform it into a hypertable with `create_hypertable`:
 --   using the values in the `time` column.
 
 SELECT create_hypertable('conditions', 'time');
-
--- OR you can additionally partition the data on another
---   dimension (what we call 'space partitioning').
--- E.g., to partition `location` into 4 partitions:
-
-SELECT create_hypertable('conditions', 'time', 'location', 4);
 ```
 
+You can additionally partition data on another dimension (what we call
```

Comment:
I think we should probably still keep the example because the purpose of the documentation here is to explain how you use the API.

Reply:
Done, added back that example, but after this warning text so that people don't immediately think they need it.